src/analysis/classic/warrior/arms/Guide.tsx

```diff
@@ -0,0 +1,40 @@
+import SPELLS from 'common/SPELLS/classic';
+import { maybeGetTalentOrSpell } from 'common/maybeGetTalentOrSpell';
+import { CLASSIC_EXPANSION } from 'game/Expansion';
+import { Section, useInfo } from 'interface/guide';
+import PreparationSection from 'interface/guide/components/Preparation/PreparationSection';
+import { FoundationCooldownSection } from 'interface/guide/foundation/FoundationCooldownSection';
+import { FoundationDowntimeSection } from 'interface/guide/foundation/FoundationDowntimeSection';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+import { useMemo } from 'react';
+
+const COOLDOWN_BLACKLIST = [SPELLS.BLADESTORM.id, SPELLS.SWEEPING_STRIKES.id];
+
+export default function Guide() {
+  const abilities = useInfo()?.abilities;
+  // want to exclude some aoe-only cooldowns from the cd list.
+  // these probably warrant special handling for aoe fights but I'm not certain what that should look like
```

Comment:
I usually just put the ability under `ROTATIONAL_AOE` and add a cooldown. That way you can see the cooldown in the stats list but it won't be included in the CD module for the guide.

But this way works too if that's how you want it.